### PR TITLE
chore: add a new tag for tutorial integration tests

### LINKED ISSUE #807: Add a new tag for tutorial integration tests
It would be good to add a new tutorial tag in the integration tests, to make it a convention to add a test for each new tutorial, and easily find them. Once the tag is added, we should update the relevant integration tests and also update the contribution guidelines.

### PR DESCRIPTION
**Pull request by tromai**
Closes #807 .

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -185,14 +189,49 @@ def do_something(value: bool = False) -> bool:
...
+make setup
+```
+  - Run these tests locally with:
 ```bash
 make integration-test
 ```
+2. Integration tests that are run against the container image.
+  - Prerequisites: Setup the integration test utility for docker.
+```bash
+make setup-integration-test-utility-for-docker
```
Comment (behnazh-w) [inline CONTRIBUTING.md:209]: This is not needed. `make setup` will install all the required dependencies.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -185,14 +189,49 @@ def do_something(value: bool = False) -> bool:
...
 
-Note that integration tests can take a long time to complete. Also the repositories that we clone for these tests will be stored under `output/` directory. If you do not remove/move this directory and run the pre-commit tool you might get errors.
+Note that integration tests can take a long time to complete.
 
 
+Each integration test case has a set of tags. Please follow these instructions on how a test case is tagged for our CI/CD pipeline:
+- If you want a test case to **only** run for the container image, use **only** `macaron-docker-image`.
+- If you want a test case to **only** run with the Macaron Python package, use **only** `macaron-python-package`.
+- To skip a test case, use `skip`. `skip` still has the same effect if it's used with other tags.
+- If you want to run a test case for both the Macaron Python package and the docker container, use `macaron-python-package` and `macaron-docker-image` tags.
```
Comment (behnazh-w) [inline CONTRIBUTING.md:227]:
```suggestion
- If you want to run a test case for both the Macaron Python package and the docker container, use both `macaron-python-package` and `macaron-docker-image` tags.
```

Comment (tromai) [reply CONTRIBUTING.md]: Addressed in https://github.com/oracle/macaron/pull/815/commits/bbc1a27658cc537a70e7f76838be55adbb9fd19b

File: CONTRIBUTING.md (hunk 1 above)
Comment (tromai) [reply CONTRIBUTING.md]: I have removed it in https://github.com/oracle/macaron/pull/815/commits/bbc1a27658cc537a70e7f76838be55adbb9fd19b

**Review by behnazh-w — APPROVED** [review]

**Pull request merged**